department-of-veterans-affairs/va.gov-cms
# VACMS-10766: Inline Guidance groundwork and setup

## Description

Closes #10766.
Closes #10777.

## Testing done
Tested locally.

## Screen Shots
These are for the alt text inline guidance.
![CleanShot 2022-09-29 at 11 35 42](https://user-images.githubusercontent.com/109987005/193114912-3df76237-8b87-4131-a685-80c374ead35d.png)
![CleanShot 2022-09-29 at 11 35 31](https://user-images.githubusercontent.com/109987005/193114919-6de3cb6c-b7c5-433a-a6c1-fde4ffc80f2a.png)

This is for the Field Group widget.
![CleanShot 2022-09-30 at 09 58 24](https://user-images.githubusercontent.com/109987005/193320276-e941a254-1061-40d4-9116-07b6f75f5b45.png)



## QA steps
Got to Content -> Media -> Add Image and and and image. You should see the guidance. This can also be seen when you add a Story. Go To Content -> Add Content -> Story. Scroll down until you find the media section, and you will see the Add Media button. Add an image, and you should see the Guidance in the Add Image modal. 

I also added a Field Group widget. It will require working with the twig component for inline guidance to actually display the inline guidance on a content type by content type basis. See `docroot/themes/custom/vagovclaro/templates/field-group-html-element--node-banner-form--group-paths.html.twig` for an example. 
### Select Team for PR review
